### DISCUSSION TITLE
watch image in browser

### DISCUSSION THREAD

**Discussion by kalle07** [vladmandic/sdnext#3058] [Q&A]
if an image is fully calculated and i "klick" on that i see it in full screen, but with the whole prompt below 1/4 of the image is behind that, how i can disable this "prompt view"?

**Comment by vladmandic** [comment]
setting to disable that is in dev branch and will be part of the next release.

**Reply by kalle07** [reply]
oh wahooo , thx
and fast answer  ;)
